app/waf/src/controllers/declaration.controller.ts

```diff
@@ -333,4 +342,60 @@ export class DeclarationController extends BaseController {
       .declarations(applicationId)
       .delete({and: [{id: declarationId}, {tenantId: await this.tenantId}]});
   }
+
+  // TODO: not sure how to name the API path. It's time for us to discuss all of them again.
+  @post(prefix + '/declarations/{declarationId}/deploy', {
```

Comment:
It conflicts with PR #149. And we need to vote resource uri first.

Reply:
Done, changed it.